src/app/dataValidator/helpers/dataLoader/readdirSync.test.js

```diff
@@ -18,24 +21,25 @@ const expectMethodToBeCalledTimes = (
 
 describe('readdirSync helper', () => {
   it('should call readScenario for every file in the /data directory', () => {
-    const readScenarioSpy = jest.spyOn(readScenario, 'readScenario');
+    readScenarioSpy = jest.spyOn(readScenario, 'readScenario');
```

Comment:
This spy isn't used outside of the scope of this test, does it need to be scoped to the full file?